### ISSUE TITLE
Update of data not working

### ISSUE THREAD

**Issue by ballonchef** [davidrapan/ha-solarman#94]
**Describe the bug**
I moved from solarman to ha-solarman. i can retrieve data from my inverter, but the data does not update automatically. Only after i press "reload" in the intregration manually.
With the solarman integration, the automatic update worked well.

**Attach the debug log**
I don't know how to insert a debug log.

**To Reproduce**
-

**Expected behavior**
Data should automatically update every 10 seconds

**Screenshots**
![Bildschirmfoto vom 2024-08-17 23-31-11](https://github.com/user-attachments/assets/61b496b6-1fd5-4571-96d5-f8826699be89)

Here you can see that the sensor does not update automatically, only when I reload manually.
**Metadata:**
Version: [e.g. v24.08.14]

**Comment by davidrapan** [comment]
Duplicate of #91

**Comment by davidrapan** [comment]
Hi @ballonchef Try update to [v24.08.17](https://github.com/davidrapan/ha-solarman/releases/tag/v24.08.17) and see if it won't fix your issue.

**Comment by davidrapan** [comment]
If not here is how to enable debug logging:
![350130101-589ced82-4953-48d8-8ea3-683dff670836](https://github.com/user-attachments/assets/90e8d626-144a-449e-8ac0-8be064c62653)

**Comment by ballonchef** [comment]
Thanks for the very quick reply!! I'll update and report.

**Comment by ballonchef** [comment]
The update to 

> Hi @ballonchef Try update to [v24.08.17](https://github.com/davidrapan/ha-solarman/releases/tag/v24.08.17) and see if it won't fix your issue.

The update to 24.08.17 solved the issue. Thanks a lot, again!!

**Comment by davidrapan** [comment]
Excellent! You're very welcome!

**Comment by PV-Joe** [comment]
@ballonchef

Nice to see you over here.
Hope you are happy with the fast update interval and the other advantages of this integration.

Happy monitoring ☀️
